WEAPON/GetWeapontypeGroup.md

```diff
@@ -23,3 +23,14 @@ Hash GET_WEAPONTYPE_GROUP(Hash weaponHash);
 * **Heavy Weapon**: 2725924767
 * **Throwables**: 1548507267
 * **Misc**: 4257178988
+
```

Comment:
Hey there, could you explain what your additions do? The function returns a `Hash`, which is a number, i.e. `2725924767`, which does correspond to the `Heavy Weapon` group. Not sure what the intention is here, but thank you for your contribution anyway 😃